addons/web/static/tests/core/commands/command_palette_tests.js

```diff
@@ -1547,3 +1547,53 @@ QUnit.test("generate new session id when opened", async (assert) => {
     await nextTick();
     assert.equal(lastSessionId, 1);
 });
+
+QUnit.test("checks that href is correctly used", async (assert) => {
+    await mount(TestComponent, target, { env });
+    const providers = [
+        {
+            namespace: "@",
+            provide: () => [
+                {
+                    name: "Command with link",
+                    action: () => {
+                        assert.step("action");
+                    },
+                    href: "https://www.odoo.com",
+                },
+                {
+                    name: "Command2",
```

Comment:
Can we rename "Command2" and the variable "command2" here below to something more meaningful?
"Command without link" maybe?